libnmstate/schema.py

```diff
@@ -200,6 +200,13 @@ class Ethernet(object):
     FULL_DUPLEX = 'full'
     HALF_DUPLEX = 'half'
 
+    SR_IOV = 'sr-iov'
+
+    class SRIOV(object):
+        CONFIG_SUBTREE = 'sr-iov'
```

Comment:
AFAIU the LinuxBridge example, this constant should be a member of the Ethernet class.

Reply:
Right, done! Thanks